### ISSUE TITLE
setup.py fails with UnicodeEncodeError

### ISSUE THREAD

**Issue by spwhitt** [scopatz/xonsh#90]
When running setup.py install:

```
Traceback (most recent call last):
  File "setup.py", line 114, in <module>
    main()
  File "setup.py", line 49, in main
    print(logo)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 28-31: ordinal not in range(128)
```

**Comment by mbollmann** [comment]
This has been fixed already in version 0.1.2; my guess is you're using Python 2, while xonsh requires Python 3.4+

**Comment by spwhitt** [comment]
Turns out to be an issue with locales local to my machine. Closing.
